src/ui/screens/home/controller/ConnectionStability.qml

```diff
@@ -157,7 +157,7 @@ Item {
             //: missing, asking them to check their connection.
             text: qsTrId("vpn.connectionStability.checkConnection")
             color: "#FFFFFF"
-            opacity: 0.8
+            // opacity: 0.8
```

Comment:
We've had grief with transparency on Android before so this is likely the culprit

Reply:
commenting out the opacity didn't fix this 😕